pkg/draft/pack/pack.go

```diff
@@ -61,16 +57,22 @@ func (p *Pack) SaveDir(dest string) error {
 		return err
 	}
 
-	// save Dockerfile
-	dockerfilePath := filepath.Join(dest, DockerfileName)
-	exists, err := osutil.Exists(dockerfilePath)
-	if err != nil {
-		return err
-	}
-	if !exists {
-		if err := ioutil.WriteFile(dockerfilePath, p.Dockerfile, 0644); err != nil {
+	// save the rest of the files
```

Comment:
The line above (chartutil.SaveDir(p.Chart, chartPath)) will save all the p.Chart.Files. 
However it will fail if p.Chart.Files contains a directory (e.g. <pack>/chart/directory/file.txt), as it does not create the missing directories.

I submitted the following PR to address this issue: https://github.com/kubernetes/helm/pull/3567

Reply:
thanks for pointing that out @jumal!

Reply:
k, I've merged that PR upstream. In order to take advantage of that, we'll need to bump helm in Gopkg.toml to master. Are you comfortable with doing that?

Reply:
oh wait this is my PR, not yours! ha :)